<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RouteDialogBase</class>
 <widget class="QDialog" name="RouteDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>383</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MusE: Routing</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox4">
     <property name="title">
      <string>Add Route</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="margin">
       <number>4</number>
      </property>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QGroupBox" name="groupBox2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Source:</string>
          </property>
          <layout class="QVBoxLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>2</number>
           </property>
           <item>
            <widget class="QListWidget" name="newSrcList"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Destination:</string>
          </property>
          <layout class="QVBoxLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>2</number>
           </property>
           <item>
            <widget class="QListWidget" name="newDstList"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QToolButton" name="connectButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>connect source to destination</string>
        </property>
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox1">
     <property name="title">
      <string>Current Routes</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="margin">
       <number>4</number>
      </property>
      <item>
       <widget class="QTreeWidget" name="routeList">
        <column>
         <property name="text">
          <string>Source</string>
         </property>
         <property name="clickable">
          <bool>true</bool>
         </property>
         <property name="resizable">
          <bool>true</bool>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Destination</string>
         </property>
         <property name="clickable">
          <bool>true</bool>
         </property>
         <property name="resizable">
          <bool>true</bool>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="removeButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>remove selected route</string>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
